react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    if (this.state.isClicked) {
+      this.setState({ isClicked: false });
+    } else {
+      this.setState({ isClicked: true });
+    }
+  }
+
+  render() {
+    if (!this.state.isClicked) {
+      return (
+        <div className='container background-on'>
+          <div className='on'>
+            <button className='switch' onClick={this.handleClick}></button>
+          </div>
+          <p>ON</p>
+        </div>
+      );
+    } else {
+      return (
+        <div className='container background-off'>
+          <div className='off'>
+            <button className='switch-off' onClick={this.handleClick}></button>
+          </div>
+          <p>OFF</p>
+        </div>
+      );
+    }
+  }
```

Comment:
This will be hard to maintain and will be prone to bugs and inconsistencies. Here is a better approach:
```JSX
let status = 'off';

if (!this.state.isClicked) {
  status = 'on';
} 

return (
  <div className={`container background-${status}`}>
    <div className={status}>
      <button className={`switch-${status}`} onClick={this.handleClick}></button>
    </div>
    <p>{status.toUpperCase()}</p>
  </div>
);
```